Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -403,8 +403,12 @@ extension ActivityStreamPanel {
         let siteBGColor = topSiteItemCell.contentView.backgroundColor
 
         let site = self.topSitesManager.content[indexPath.item]
+        let domainURL = NSURL(string: site.url)?.domainURL().absoluteString ?? site.url
```

Comment:
Looking through the `domainURL` implementation, I see a couple of issues:
1. We use `normalizedHost`, which drops the www/mobile/m prefix for the domain. This is fine for display, but I doubt we want to do this for the actual URL used for navigation.
1. We drop the port in `domainURL`. Not a huge issue since non-80 ports are uncommon, but this will make unit tests more difficult.